optfiles/linux_amd64_ifort+mpi_stampede2_skx

```diff
@@ -21,6 +21,8 @@ NOOPTFLAGS='-O1 -fp-model precise'
 
 FFLAGS="$FFLAGS -W0 -WB -convert big_endian -assume byterecl"
 FFLAGS="$FFLAGS -fPIC"
+FFLAGS="$FFLAGS -mcmodel=large -shared-intel"
+CFLAGS="$CFLAGS -mcmodel=large -shared-intel"
```

Comment:
I don't have any of these options included in my `CFLAGS`

Reply:
I'll look into why I had this in `CFLAGS` as well, will get back to it

Reply:
I'm still not entirely sure, but I would suggest we keep it in, because something similar is in [MITgcm/tools/build_options/linux_amd64_ifort+impi](https://github.com/MITgcm/MITgcm/blob/448d225072cf8aa5e45b6d274f6b15e079823f28/tools/build_options/linux_amd64_ifort%2Bimpi#L42-L44) and that's good enough for me...